docs/src/modules/drawerDataGroups.js

```diff
@@ -0,0 +1,457 @@
+import { Button, Card, Drawer, Header, Icon, Input, TitleBar } from 'react-cm-ui';
+import DrawerSubNavigation from './drawerSubNavigation.js';
+import Highlighter from '../app/highlighter.js';
+import Main from '../app/main.js';
+import React from 'react';
+import TableProps from '../app/tableProps.js';
+
+const drawerDataGroupsSample = `import { Button, Drawer, InfoBar } from 'react-cm-ui';
+import React from 'react';
+
+export default class DrawerDataGroupsSample extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            isDrawerOpen: false,
+        };
+
+        this._onDrawerToggle = this._onDrawerToggle.bind(this);
```

Comment:
No more leading underscores.  Please adapt this file to the new ESLINT rules and style preferences.

Reply:
Also note: no more multiple class components per file per the new ESLINT rules, so please split this into separate files.

Reply:
Fixed.